documentation/docs/infrastructure/vault.md

```diff
@@ -1,25 +1,47 @@
 # Vault for Pipeline Secrets
 
-Project "Piper" also supports fetching your pipeline secrets directly from [Vault](https://www.hashicorp.com/products/vault).
-Currently Vault's key value engine is supported in version 1 and 2, although we recommend version 2 since it supports versioning of secrets
+Project "Piper" supports fetching your pipeline secrets directly from [Vault](https://www.hashicorp.com/products/vault).
+Currently, Vault's key value engine is supported in version 1 and 2, although we recommend version 2 since it supports
+the versioning of secrets
 
 Parameters that support being fetched from Vault are marked with the Vault Label in the Step Documentation.
 
 ![Vault Label](../images/parameter-with-vault-support.png)
 
-## Vault Setup
+## Authenticating Piper to Vault
 
-The first step to store your pipeline secrets in vault, is to enable a the [Key-Value Engine](https://www.vaultproject.io/docs/secrets/kv/kv-v2). And then create a policy which grants read access to the key value engine.
-For Piper to authenticate against Vault, [AppRole](https://www.vaultproject.io/docs/auth/approle) authentication must be enabled in your Vault instance.
-You have to [create an AppRole Role](https://www.vaultproject.io/api-docs/auth/approle#create-update-approle) for Piper and assign it the necessary policies.
+Piper currently supports Vault's `AppRole` and `Token` authentication. However, `Token` authentication is recommended
+since Piper is able to regularly rotate the SecretID, which is not possible with a Token.
```

Comment:
The last sentence grammatically sounds like there is a mistake. "Use token because ... is not possible for token" - if this is not only irritating me, it might be better to rephrase/explain the recommended auth method.

Reply:
whoops 😆 
good catch

Reply:
Done